carbonmark-api/src/utils/helpers/utils.ts

```diff
@@ -220,20 +220,11 @@ const getPoolPrice = async (
 
 /** @todo refactor this */
 export async function calculatePoolPrices(fastify: FastifyInstance) {
-  let decimals: number;
-  if (process.env.VERCEL_ENV == "production") {
-    decimals = 1e6;
-  } else {
-    decimals = 1e18;
-  }
```

Comment:
@Atmosfearful @Najada do you happen to remember why we had this decimals if case here?

Also in carbonmark:

https://github.com/KlimaDAO/klimadao/blob/ea2d095d469677ed2b48a8615372a90f31323d17/lib/utils/getTokenDecimals/index.ts#L6

Reply:
Yea we had configured staging & previews to use testnet (mumbai), and we didn't have a 6 decimal ERC20 token on mumbai, so we used an 18 decimal one that najada already had.